### ISSUE TITLE
Suggest: changing the package go ioutil was deprecated support <1.16 to new native package io.

### ISSUE THREAD

**Issue by Paulo-Lopes-Estevao** [postmanlabs/postman-code-generators#711]
**Is your feature request related to a problem? Please describe.**
Package go `ioutil` was deprecated support <1.16 the same features are now in the `io` package and with more features and support.

**Describe the solution you'd like**
```go
import (
  "fmt"
  "strings"
  "net/http"
  "io/ioutil"
)

ioutil.ReadAll(res.Body)
```

to
```go
import (
  "fmt"
  "strings"
  "net/http"
  "io"
)

io.ReadAll(res.Body)
```

**Additional context**
<img width="1504" alt="Screenshot 2023-06-21 at 10 32 58" src="https://github.com/postmanlabs/postman-code-generators/assets/66704496/491b4df5-2a39-4eaa-8798-6ac596e8be1c">

**Comment by kaifkh20** [comment]
io.ReadAll is only available ^1.16. 

So basically when I try to change it two of the test cases fail.
For now I think ioutil.ReadAll is fine as it will also work for version below 1.16.

**Comment by Paulo-Lopes-Estevao** [comment]
I understand it was a suggestion so far we have not broken compatibility and it is still working.
I'll keep an eye on the future change anything mention.
